src/StreetNameRegistry/Municipality/Municipality.cs

```diff
@@ -1,12 +1,10 @@
 namespace StreetNameRegistry.Municipality
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
     using Be.Vlaanderen.Basisregisters.AggregateSource;
     using Be.Vlaanderen.Basisregisters.AggregateSource.Snapshotting;
     using Events;
     using Exceptions;
+    using System.Collections.Generic;
```

Comment:
there's a config in rider which puts `System` usings first

Reply:
Fixed.